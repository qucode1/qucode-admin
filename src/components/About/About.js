import React from 'react'

import styles from './About.css'

const About = () => (
  <div>
    <h1 className={styles.title}>Hi, I am Yannick</h1>
    <p className={styles.content}>This section is all about me, yay!</p>
  </div>
)

export default About
