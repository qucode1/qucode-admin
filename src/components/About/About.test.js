import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import About from './About'

describe('<About />', () => {
  it('should render title and paragraph as expected', () => {
    const wrapper = shallow(<About />)
    const tree = toJson(wrapper)
    expect(tree).toMatchSnapshot()
  })
})
