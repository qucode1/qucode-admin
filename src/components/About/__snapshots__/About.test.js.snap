// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render title and paragraph as expected 1`] = `
<div>
  <h1
    className="title"
  >
    Hi, I am Yannick
  </h1>
  <p
    className="content"
  >
    This section is all about me, yay!
  </p>
</div>
`;
